auth.go

```diff
@@ -36,13 +37,27 @@ func getRegion(registry string) (string, error) {
 	return ecrRegion, nil
 }
 
-func getAuthFromIRSA(ctx context.Context, ref registry.Reference) (authn.AuthConfig, error) {
+func filterAWSImages(image string) bool {
```

Comment:
@vishal-chdhry - I am not sure why this is necessary. We should be able to support images signed with notation and AWS signer but stored in registries other than ECR.

Reply:
@JimBugwadia It is possible that someone is using an image like busybox/nginx for init containers, notation will try to verify those and fail

Besides, verification of images outside of ECR + AWS Signer context can be done using kyverno and I dont think it will be any helpful to try to verify them here as this focusses on AWS signer and ECR

Reply:
@vishal-chdhry the question is on images signed by notation and the signer binary, but stored in registries other than ECR. Those cannot be verified without the extension.

Reply:
@JimBugwadia , we can add a field for matching image references like to filter images instead of making it strict